src/api/index.js

```diff
@@ -282,6 +282,24 @@ export function getPipelineResource({ name, namespace }) {
   return get(uri);
 }
 
+export function getConditions({ filters = [], namespace } = {}) {
+  const uri = getTektonAPI(
+    'conditions',
+    { namespace, version: 'v1alpha1' },
+    getQueryParams(filters)
+  );
+  return get(uri).then(checkData);
+}
+
+export function getCondition({ name, namespace }) {
+  const uri = getTektonAPI('conditions', {
+    name,
+    namespace,
+    version: 'v1alpha1'
```

Comment:
Interesting this is still at v1alpha1

Reply:
yeah Condition is still alpha, had to double-check: https://github.com/tektoncd/pipeline/blob/524e467f7e3bbd0ccf6a5a7ce666dd2e48166afd/config/300-condition.yaml#L37

> v1beta1 is gonna be the first beta API supported by Tekton Pipeline. It contains `Pipeline`, `PipelineRun`, `Task`, `ClusterTask` and `PipelineRun` objects. **`Condition` and `PipelineResource` are staying in alpha state**. This means they can be used in the beta object *but* the won't be supported.